QC.wdl

```diff
@@ -2,19 +2,16 @@ version 1.0
 # Copyright 2018 Sequencing Analysis Support Core - Leiden University Medical Center
 
 import "QualityReport.wdl" as QR
-import "AdapterClipping.wdl" as AC
-import "tasks/biopet/seqstat.wdl" as seqstat
-import "tasks/biopet/biopet.wdl" as biopet
+import "tasks/cutadapt.wdl" as cutadapt
 import "tasks/common.wdl" as common
 
 workflow QC {
     input {
-        FastqPair reads
+        File read1
+        File? read2
         String outputDir
         Boolean alwaysRunAdapterClipping = false
-        String sample
-        String library
-        String readgroup
+        Int minimumReadLength = 2 # Choose 2 here to compensate for cutadapt weirdness. I.e. Having empty or non-sensical 1 base reads.
```

Comment:
2 base reads are pretty useless as well.

Reply:
I know. But we have to draw a line somewhere. Minimumreadlength = 1 perhaps? (This was the default at first, but may have crashed some tools)

Reply:
2 is fine I guess.